src/test/java/org/broadinstitute/hellbender/GATKBaseTest.java

```diff
@@ -30,6 +30,7 @@
 
     public static final String packageRootTestDir = publicTestDir + "org/broadinstitute/hellbender/";
     public static final String toolsTestDir = packageRootTestDir + "tools/";
+    public static final String exampleTestDir = toolsTestDir + "examples/";
```

Comment:
Someday it would be nice to modernize this code by using `File` (or, even better, `Path`) for all filesystem references. Then we can replace string concatenation in code like this `Path.resolve()`.

Also, this should use all uppercase for static finals like the rest of our code does.

Reply:
Not all of the paths in this class are local files, though (some are GCS URIs), so we don't want to switch to `File` here. Ultimately we plan to unify around a "GATK URI" class that can handle all supported inputs/outputs to the GATK (including NIO `Path`-supported filesystems such GCS, as well as types not supported via NIO such as `gendb://`): https://github.com/broadinstitute/gatk/issues/958. Until this materializes continued use of `String` is the recommended practice, for now.